src/main/java/com/github/dockerjava/jaxrs/DockerCmdExecConfig.java

```diff
@@ -0,0 +1,62 @@
+package com.github.dockerjava.jaxrs;
+
+import java.io.Serializable;
+
+/**
+ * Serializable object that store options for {@link com.github.dockerjava.jaxrs.DockerCmdExecFactoryImpl}
+ *
+ * @author Kanstantsin Shautsou
+ */
+public class DockerCmdExecConfig implements Serializable {
+    private static final long serialVersionUID = 1L;
```

Comment:
Every newly added file should have `1L` generated must be extracted only for already used classes.